CHANGELOG.md

```diff
@@ -0,0 +1,125 @@
+# Changes
+
+- 0.27.0 - Improve MD011/MD013/MD022/MD031/MD032/MD033/MD034/MD040/MD043/MD051/
+           MD053, generate/separate documentation, improve documentation, update
+           dependencies.
+- 0.26.2 - Improve MD037/MD051/MD053.
+- 0.26.1 - Improve MD051.
+- 0.26.0 - Add MD051/MD052/MD053 for validating link fragments & reference
+           links/images & link/image reference definitions (MD053 auto-fixable),
```

Comment:
This indent style wastes a lot of space. Looks like it wants to be a table instead of a list? :S

Reply:
If we're considering changing the formatting, how about using subheaders for each version? That allows users to anchor link to them individually:

```
## 0.26.1

...

## 0.26.0

...

```

That's how many autogenerated changelogs work. Example generated by [@release-it-plugins/lerna-changelog](https://github.com/release-it-plugins/lerna-changelog) that I like: https://github.com/ember-cli/eslint-plugin-ember/blob/master/CHANGELOG.md#v1120-2022-10-27

Reply:
I like this approach!

To go further, each comma-separated item in the entry for a version should be its own bullet point in a list:

```markdown
## 0.23.0

- Improved MD123
- Updated dependencies
```

Also, what do you think about using `###` for patch changes to make them distinct?

BTW, I'm happy making these changes since this has grown beyond what you signed up for. :)

Reply:
I updated to the subheaders and dasherized lists format.

I personally don't think `###` is necessary for patch versions. If the latest release is a patch release, then the headings would go from `#` to `###` which violates [MD001](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md001). And it would be inconsistent unless we plan distinct treatment for major version headers as well.